packages/expo-notifications/src/Tokens.types.ts

```diff
@@ -62,3 +62,53 @@ export interface ExpoPushToken {
    */
   data: string;
 }
+
+export interface ExpoPushTokenOptions {
+  /**
+   * Endpoint URL override.
+   */
+  baseUrl?: string;
+  /**
+   * Request URL override.
+   */
+  url?: string;
+  /**
+   * Request body override.
+   */
+  type?: string;
+  // @missingDocs
+  deviceId?: string;
+  /**
+   * Makes sense only on iOS, where there are two push notification services: "sandbox" and "production".
+   * This defines whether the push token is supposed to be used with the sandbox platform notification service.
+   * Defaults to [`Application.getIosPushNotificationServiceEnvironmentAsync()`](/application/#applicationgetiospushnotificationserviceenvironmentasync)
+   * exposed by `expo-application` or `false`. Most probably you won't need to customize that.
+   * You may want to customize that if you don't want to install `expo-application` and still use the sandbox APNs.
+   * @platform ios
+   */
+  development?: boolean;
+  /**
+   * @deprecated Use `projectId` instead.
+   * The ID of the experience to which the token should be attributed.
+   * Defaults to [`Constants.manifest.id`](/constants/#constantsmanifest) exposed by `expo-constants`.
+   * When building with EAS Build, or in the bare workflow, **this is required** and you must provide a value which takes the shape `@username/projectSlug`,
+   * where `username` is the Expo account that the project is associated with, and `projectSlug` is your [`slug`](/config/app.mdx#slug) from Expo config.
```

Comment:
Can we modify this by moving the deprecated label below the description such that the description of the field is not included in the deprecation notice.

Update:

```ts
 /**
   * The ID of the experience to which the token should be attributed.
   * Defaults to [`Constants.manifest.id`](/constants/#constantsmanifest) exposed by `expo-constants`.
   * When building with EAS Build, or in the bare workflow, **this is required** and you must provide a value which takes the shape `@username/projectSlug`,
   * where `username` is the Expo account that the project is associated with, and `projectSlug` is your [`slug`](/config/app.mdx#slug) from Expo config.
   *
   * @deprecated Use `projectId` instead.
   */
  experienceId?: string;
```

End result:

<img width="822" alt="CleanShot 2023-02-07 at 17 50 58@2x" src="https://user-images.githubusercontent.com/10234615/217243765-9756ffda-929a-442b-bf30-f13d645d548a.png">

Reply:
Sure, that was for sure not intended. Feel free to update this PR branch if you want.